### PR TITLE
Cleanup and update SECURITY.md contents and maintainer's email IDs.

### PR DESCRIPTION
**Pull request by gapisback**
Minor cleanup of `SECURITY.md` and updates to email-IDs of maintainers.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
-jmanferdelli@vmware.com, yel@vmware.com, agurajada@vmware.com, johnmanferdelli@hotmail.com
+John Manferdelli    - jmanferdelli@vmware.com, johnmanferdelli@hotmail.com, jlmucbmath@gmail.com
+Ye Li               - yel@vmware.com
+Aditya Gurajada     - agurajada@vmware.com, gapisback@gmail.com
+Radoslav Gerganov   - rgerganov@vmware.com, rgerganov@gmail.com
```
Comment (gapisback) [inline MAINTAINERS.md:4]: @jlmucb @yelvmw @rgerganov -- Please cross-check these mail IDs I'm listing here.

Some of your GitHub check-ins are associated with non-VMware mail IDs. So, I'm listing those in this file.

We need to finalize ASAP.

File: SECURITY.md [hunk 2]
```diff
@@ -1,17 +1,36 @@
...
+
+## Supported Versions
 
-| Version | Supported          |
-| ------- | ------------------ |
-| 5.1.x   | :white_check_mark: |
-| 5.0.x   | :x:                |
-| 4.0.x   | :white_check_mark: |
-| < 4.0   | :x:                |
+The Certifier Framework is in late-stage Beta development phase. No major releases are supported, yet.
```
Comment (gapisback) [inline SECURITY.md:32]: @jlmucb - I simplified this releases section as we don't have any releases, yet.

**Comment by gapisback** [comment]
HI, @jlmucb @yelvmw @rgerganov - Given that we are getting the nod on CCC proposal, I think ...  We NEED to finalize these changes asap.

The `SECURITY.md` was pending on my plate for weeks. 

All minor cleanup --please give this a quick look.

File: MAINTAINERS.md (hunk 1 above)
Comment (rgerganov) [reply MAINTAINERS.md:4]: LGTM, thanks

**Review by rgerganov — APPROVED** [review]

**Review by jlmucb — APPROVED** [review]

**Review by yelvmw — APPROVED** [review]
LGTM

**Pull request merged**